composer.json

```diff
@@ -22,12 +22,13 @@
         }
     ],
     "require": {
-        "php": "^7.4 || ^8.0",
+        "php": "^8.0",
         "illuminate/contracts": "^8.0",
-        "spatie/data-transfer-object": "^2.5.0",
+        "spatie/data-transfer-object": "^3.0",
         "thecodingmachine/safe": "^1.3"
     },
     "require-dev": {
+        "mockery/mockery": "^1.4",
```

Comment:
What was this needed for?

Reply:
@jessarcher Fixes deprecation error in `prefer-lowest` workflow: https://github.com/jessarcher/laravel-castable-data-transfer-object/runs/2265255816.